src/common.h

```diff
@@ -102,8 +102,8 @@ enum class Result {
   Error,
 };
 
-#define WABT_SUCCEEDED(x) ((x) == Result::Ok)
-#define WABT_FAILED(x) ((x) == Result::Error)
+#define WABT_SUCCEEDED(x) ((x) == ::wabt::Result::Ok)
+#define WABT_FAILED(x) ((x) == ::wabt::Result::Error)
```

Comment:
It's probably not needed. It just makes sure that you're accessing the global namespace, in case `wabt::Result::Ok` was defined in some other namespace.